.jenkins/pipelines/OpenEnclave/intel/tests/ubuntu.Jenkinsfile

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+properties(
+    [
+        buildDiscarder(
+            logRotator(
+                artifactDaysToKeepStr: '90',
+                artifactNumToKeepStr: '180',
+                daysToKeepStr: '90',
+                numToKeepStr: '180'
+            )
+        ),
+        [$class: 'JobRestrictionProperty'],
+        parameters(
+            [
+                string(name: "DCAP_URL", description: "Intel DCAP Package URL"),
+                string(name: "PSW_URL", description: "Intel PSW Package URL"),
+                string(name: "OECI_LIB_VERSION", defaultValue: 'release-pipeline', description: 'Version of OE Libraries to use'),
+                string(name: "OE_RELEASE_VERSION", description: "Open Enclave Release Version"),
+                choice(name: "OE_PACKAGE", defaultValue: "open-enclave", choices: ["open-enclave", "open-enclave-hostverify"], description: "Open Enclave package type to install"),
+                choice(name: "RELEASE_SOURCE", defaultValue: "Github", choices: ["Azure", "GitHub"], description: "Source to download the OE Release from")
+            ]
+        )
+    ]
+)
+
+library "OpenEnclaveJenkinsLibrary@${params.OECI_LIB_VERSION}"
```

Comment:
This is more of a preference, but it is best to put this at the top so we can immediately see a library is being used and understand its implications throughout the Jenkinsfile

Reply:
Same applies to the other Jenkinsfile